src/main.zig

```diff
@@ -107,13 +107,15 @@ const Command = struct {
     }
 
     pub fn start(
-        allocator: mem.Allocator,
+        arena: *std.heap.ArenaAllocator,
```

Comment:
Can we not keep the standard `mem.Allocator` interface here?

Reply:
We need to know that it's an arena in order to count the allocations. The count needs to happen in this function so it can be between init and running. 

This is an internal function in main.zig which is only called from one place, so it doesn't seem like a loss of modularity. 

I initially did the count inside Replica.open, but then the replica needs to know that it's using an arena allocator.